samcli/lib/package/packageable_resources.py

```diff
@@ -599,7 +600,7 @@ def get_property_value(self, resource_dict):
 
 class GraphQLApiSchemaResource(ResourceZip):
     RESOURCE_TYPE = AWS_SERVERLESS_GRAPHQLAPI
-    PROPERTY_NAME = RESOURCES_WITH_LOCAL_PATHS[RESOURCE_TYPE][0]
+    PROPERTY_NAME = graphql_api.SCHEMA_ARTIFACT_PROPERTY
```

Comment:
nit: Can we import anything we need from `graphql_api` directly instead of importing the module?

Reply:
Is the style I used is against SAM CLI practice? I import everything from that file anyway. But the main reason, I think that `graphql_api.SCHEMA_ARTIFACT_PROPERTY` and `graphql_api.find_all_paths_and_values(self.PROPERTY_NAME, resource_dict)` is easier to read then simply `SCHEMA_ARTIFACT_PROPERTY` and `find_all_paths_and_values` which show less "belonging". But if it's against the style, I can remove it. It's up to you

Reply:
I think it's fine since `graphql_api` is not a huge module. And +1 for the point of "belonging".